hera_qm/xrfi.py

```diff
@@ -410,13 +482,13 @@ def flag(uvf_m, nsig_p=6., nsig_f=None, nsig_t=None, avg_method='quadmean'):
 
     # Pixel flagging
     if nsig_p is not None:
-        uvf_f.flag_array[uvf_m.metric_array > nsig_p] = True
+        uvf_f.flag_array[uvf_m.metric_array >= nsig_p] = True
 
     if uvf_m.type == 'baseline':
         if nsig_f is not None:
             # Channel flagging
             d = avg_f(uvf_m.metric_array, axis=(0, 1, 3), weights=uvf_m.weights_array)
```

Comment:
`d` is a confusing variable name. I want it to be data, not an averaged waterfall over some axes.

Reply:
It's just a throw-away. I don't want to overthink what to call a temporary variable that has no use outside this if statement.